### PR TITLE
V3: Adds ruler inspector ui

### PR DESCRIPTION
**Pull request by eireland**
Adds palette for ruler inspector

### CODE REVIEW

**Review by kswenson — CHANGES_REQUESTED** [review]
👍 Generally looks good -- I made some suggestions for simplifying the code.

File: v3/src/components/graph/components/inspector-panel/graph-measure-panel.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      description: `You clicked on ${measure} ${val}`,
+      status: 'success',
+      duration: 5000,
+      isClosable: true,
+    })
+  }
+
+  const paletteTop = graphModel.plotType === "casePlot" || graphModel.plotType === "dotChart"
+                      ? 50
+                      : 0
```
[the remark this answers is not in the document]
Comment (eireland) [reply v3/src/components/graph/components/inspector-panel/graph-measure-panel.tsx:55]: Yes. We'll have to work on this again when we have movable components

File: v3/src/components/graph/components/inspector-panel/graph-measure-panel.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+import { InspectorPalette } from "../../../inspector-panel"
+import ValuesIcon from "../../../../assets/icons/icon-values.svg"
+
+import "./point-format-panel.scss"
+
+interface IProps {
+  graphModel: IGraphModel
+}
+
+export const GraphMeasurePalette = ({graphModel}: IProps) => {
```
Comment (kswenson) [inline v3/src/components/graph/components/inspector-panel/graph-measure-panel.tsx:14]: I just noticed that the component is `GraphMeasurePalette` while the files are `graph-measure-panel.*`. Seems like they should be consistent.

File: v3/src/components/graph/components/inspector-panel/graph-measure-panel.tsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  const paletteTop = graphModel.plotType === "casePlot" || graphModel.plotType === "dotChart"
+                      ? 50
+                      : 0
+
+  return (
+    <InspectorPalette
+      title={t("DG.Inspector.values")}
+      Icon={<ValuesIcon />}
+      paletteTop={paletteTop}
+      buttonLocation={graphModel.plotType === "casePlot" || graphModel.plotType === "dotChart" ? 25 : 75}
```
[the remark this answers is not in the document]
Comment (kswenson) [reply v3/src/components/graph/components/inspector-panel/graph-measure-panel.tsx:62]: What's needed is a way to encapsulate all of the plot-specific information in one place, but that can wait for another day.

**Review by kswenson — APPROVED** [review]
👍 Looks good -- just one suggestion about consistency between file names and component names that should be addressed.

**Pull request merged**